VultisigApp/VultisigApp/Services/RpcService.swift

```diff
@@ -57,6 +57,8 @@ class RpcService {
                     || message.contains("nonce too high")
                     || message.contains("transaction already exists")
                     || message.contains("many requests for a specific RPC call")
+                    || message.contains("already")
```

Comment:
why wee need "message.contains("already")" check?

Reply:
In case the message returns "already .... anything."